src/iris/api.py

```diff
@@ -1825,6 +1910,87 @@ def on_post(self, req, resp):
         resp.set_header('Location', '/plans/%s' % plan_id)
 
 
+def count_fields_for_incidents(cursor, where_clause, fields, allowed_columns, extra_joins=''):
```

Comment:
we can also make these methods part of their respective classes e.g. Plans, Incidents, Templates unless we plan to use these elsewhere outside the context of the objects.

Reply:
done